iothub/device/src/Pipeline/MqttTransportHandler.cs

```diff
@@ -1181,10 +1176,10 @@ private void HandleTwinResponse(MqttApplicationMessageReceivedEventArgs received
             {
                 byte[] payloadBytes = receivedEventArgs.ApplicationMessage.Payload ?? Array.Empty<byte>();
 
-                if (_getTwinResponseCompletions.TryRemove(receivedRequestId, out TaskCompletionSource<GetTwinResponse> getTwinCompletion))
+                if (_pendingTwinOperations.TryRemove(receivedRequestId, out PendingMqttTwinOperation patchTwinOperation))
                 {
                     if (Logging.IsEnabled)
-                        Logging.Info(this, $"Received response to get twin request with request id {receivedRequestId}.");
+                        Logging.Info(this, $"Received response to patch twin request with request id {receivedRequestId}.", nameof(HandleTwinResponse));
```

Comment:
```suggestion
                        Logging.Info(this, $"Received response to twin request with request id {receivedRequestId}.", nameof(HandleTwinResponse));
```

We don't really know anymore if this is a response to a twin patch request or a twin get request anymore, so this can be more generic

Reply:
Fixed